### ISSUE TITLE
Accordion Block Hardcoded aria-level

### ISSUE THREAD

**Issue by esizer** [gambitph/Stackable#1001]
**Describe the bug**
The accordion block has a hardcoded ```aria-level="3"``` which can have a negative impact on accessibility when a different heading rank is more suitable.

[src/block/accordion/save.js:80](https://github.com/gambitph/Stackable/blob/e0130d414687856c59e7bda1997bf04d35694819/src/block/accordion/save.js#L80)

**To Reproduce**
1. Add accordion block
2. Change title heading rank to 2,4,5,6
3. Save post
4. See that the ```aria-level="3"``` exists despite changing heading rank.

**Expected behavior**
Either no ```aria-level``` attribute should exist or it should match the selected title heading rank.

**Comment by crapsh00t** [comment]
We just purchased the plugin and love it. But unfortunately the accessibility requirements in Canada are strict and we cannot continue to use this without this issue getting corrected ASAP. We will have to request a refund and we are really hoping you can come through here and ensure future accessibility compliance with regard to WCAG 2.1 AA guidelines.

**Comment by bfintal** [comment]
Hey @crapsh00t yes this is something we definitely want to address. For now if you need to address this asap, an alternative would be to edit the block's html and remove the `aria-label`
